templates/rejettohfs3.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>RejettoHFS3</Name>
+  <Repository>rejetto/hfs:v0.53.0</Repository>
+  <Registry>https://hub.docker.com/r/rejetto/hfs</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Project>https://github.com/rejetto/hfs/discussions</Project>
+  <Overview>Rejeto HFS Http File Server&#xD;
+&#xD;
+https://www.rejetto.com/hfs/ &#xD;
+Access via web your files directly from your disk&#xD;
+You be the server! Share files fresh from your disk, with unlimited space and bandwidth.&#xD;
+Fast! Try zipping 100GB, downloads starts immediately!&#xD;
+Share even a single file with our virtual file system, even with a different name, all without touching the real file. Present things the way you want!&#xD;
+Watch all activities in real-time.&#xD;
+Control bandwidth: decide how much to give.&#xD;
+No intermediaries: give a huge file to your friend without waiting for it to be uploaded on a server first.&#xD;
+Intelligent: HFS tries to detect problems and suggest solutions.&#xD;
+Expandable: find the right plugin, or make you own&#xD;
+&#xD;
+Easily run a web page, mange users groups and  share a file thorugh your Web Browser.&#xD;
+&#xD;
+Docker Compose Varient&#xD;
+https://github.com/bmartino1/hfs-docker&#xD;
+&#xD;
+## Other Environment HFS Variables&#xD;
+This docker image doesn't have any specific env. Every env starting with `HFS_` will be passed to HFS.&#xD;
+Read (https://github.com/rejetto/hfs/blob/main/config.md#how-to-modify-configuration) page to learn more about how envs work.&#xD;
+&#xD;
+## Docker Volumes and Mounts&#xD;
+You can mount as many volumes as you wish in docker to persist the file storage, but keep in mind that if you want to persist HFS configurations as well you **must** mount a volume that points to the `cwd` of HFS (which you can override with `HFS_CWD` env).&#xD;
+The default hfs cwd of the container is `/home/hfs/.hfs`&#xD;
+&#xD;
+## HTTPS make certs:&#xD;
+openssl req -x509 -newkey rsa:4096 -keyout /mnt/user/appdata/hfs/certs/privkey.key -out /mnt/user/appdata/hfs/certs/cert.pem -days 365 -nodes&#xD;
+</Overview>
+  <Category>Downloaders: Tools: MediaApp:Video MediaApp:Music MediaApp:Books MediaApp:Photos MediaApp:Other MediaServer:Video MediaServer:Music MediaServer:Books MediaServer:Photos MediaServer:Other Network:Web Network:FTP Network:Proxy Network:Other Status:Beta</Category>
+  <WebUI>http://[IP]:[PORT:80]/~/admin/</WebUI>
+  <Icon>https://raw.githubusercontent.com/bmartino1/hfs-docker/refs/heads/main/hfs.png</Icon>
+  <ExtraParams>--hostname HFS3 -p 8080:8080 -p 80:80 -p 443:443</ExtraParams>
```

Comment:
it is easeier to set the HFS deocker port via HFS verable and explain to use the same port in the extr parmenter i went that method on purpose. 

otherwise a template port would need ot be added for port 80 and 443 unless the HFS_Http port variapbe is changed.
same with HFS Http port 443...